src/Shared/Recruit.Vacancies.Client/Application/Validation/Fluent/FluentVacancyValidator.cs

```diff
@@ -605,38 +605,56 @@ private void ValidateAdditionalQuestions()
             When(x => !string.IsNullOrEmpty(x.AdditionalQuestion1), () =>
             {
                 RuleFor(x => x.AdditionalQuestion1)
+                    .Cascade(CascadeMode.Continue)
```

Comment:
Links to the above with the new show-all attribute on the validation tag helper, but where is the requirement for this?

Reply:
In either case, CascadeMode.Continue doesn't appear to be as necessary as I thought. It still shows all errors without setting this. So I've removed from the latest commit.